backend/sozisel/lib/sozisel_web/schema.ex

```diff
@@ -6,6 +6,7 @@ defmodule SoziselWeb.Schema do
   import_types(SoziselWeb.Schema.Types.HelloTypes)
```

Comment:
Can you get rid of all those HelloTypes, HelloMutations, HelloSUbscriptions? They were just for a showcase and we don't need them anymore.